conans/client/conan_api.py

```diff
@@ -962,13 +963,22 @@ def remote_clean(self):
         return self._cache.registry.clear()
 
     @api_method
-    def remove_locks(self):
-        self._cache.remove_locks()
+    def remove_system_reqs(self, ref):
```

Comment:
This is a ref, the conan_api shouldn't be receiving refs, but string references.
Please follow that design, at least until the conan_api is re-designed

Reply:
fixed in #5236

Reply:
Good to know!